src/app/settings/page.tsx

```diff
@@ -16,6 +19,9 @@ const Plans: Record<`did:${string}`, { name: string, limit: number }> = {
   'did:web:free.web3.storage': { name: 'Free', limit: Infinity },
 }
 
+const MAX_REFERRALS = 11
+const MAX_CREDITS = 460
```

Comment:
**suggestion**: read it from env.

Reply:
ah yea - I'm going to come back to this once we have support for actually giving people credits and displaying them - will move to config once that's done!